portfolio/src/main/webapp/script.js

```diff
@@ -34,3 +34,62 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact; 
 }
+
+const size = 30;
+var canvas;
+var ctx;
+var x = new Array(5);
+var y = new Array(5);
+
+function initSnake() {
+  canvas = document.getElementById('snakeCanvas'); // Get the canvas element by Id
+  ctx = canvas.getContext('2d'); // Canvas 2d rendering context
+  for (var z=0; z < 5; z++) {
```

Comment:
space on both sides of any binary or ternary operator: "z = 0" and elsewhere below

https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace